cmd/edgectl/main.go

```diff
@@ -123,6 +123,9 @@ func getRootCommand() *cobra.Command {
 		Long:         myHelp,
 		SilenceUsage: true, // https://github.com/spf13/cobra/issues/340
 	}
+	_ = rootCmd.PersistentFlags().Bool(
+		"no-report", false, "turn off anonymous crash reports and log submission on failure",
```

Comment:
@ark3 I added this flag as a global option to edgectl (as opposed to an install cmd flag). This is something I feel we want top-level and applicable to other sub-commands in the future... Let me know what you think :)

```
[...]

Flags:
  -h, --help        help for edgectl
      --no-report   turn off anonymous crash reports and log submission on failure

Use "edgectl [command] --help" for more information about a command.
```

Reply:
Sensible. :+1: